Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -341,6 +341,7 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
     InjectedContentVector injectedContents() const;
     bool hasInjectedContentForURL(const URL&);
     bool hasInjectedContent();
+    bool safeToInjectContent() const { return isLoaded() && m_safeToInjectContent; }
```

Comment:
Nit: Depending on how expensive `isLoaded` is, it might be more performant to put `m_safeToInjectContent` first.

Reply:
It is quick, it is just a null check.